Magento_Theme/templates/html/social-list.phtml

```diff
@@ -0,0 +1,42 @@
+<ul class="list list--horizontal list--center">
+    <li class="list__item">
+        <a class="button button--link button--icon"
+           title="<?php /* @escapeNotVerified */echo __('Facebook') ?>"
+           href="#"
```

Comment:
We should define a way how to set these links in admin area, to not write same code over and over again for every project. (if it's necessary ask for backend help)